tests/test_extractor.py

```diff
@@ -20,21 +20,29 @@ def test_no_chunks(self, tmp_path: Path):
 
     def test_one_chunk(self, tmp_path: Path):
         content = b"test file"
-        test_file = io.BytesIO(content)
+        infile = tmp_path / "infile"
```

Comment:
When you change implementation, but the feature is the same, tests should not be changed at all, which would show that the features we put there in the first place still work.

Reply:
agreed.